### PR TITLE
spatial_wms: update WMS URL

### PR DESCRIPTION
**Pull request by florisvdh**
## Description

The Geopunt URLs of OGC webservices have changed; this PR solves it for the WMS tutorial.

Note @damianooldoni @peterdesmet that I made Hugo allow raw HTML for this PR to succeed (7db836f9 - see its commit message body). This may be regarded a security issue. If someone knows how to make pandoc or rmarkdown (or whoever is doing this) not insert the resulting figures as HTML for gfm but as markdown figure references, then 7db836f9 can be reverted.

## Task list

- [X] My tutorial or article is placed in a subfolder of `tutorials/content`
- [X] The filename of my tutorial or article is `index.md`. In case of an Rmarkdown tutorial I have knitted my `index.Rmd` to `index.md` (both files are pushed to the repo). 
- [X] I have included `tags` in the YAML header (see the tags listed in the [tutorials website side bar](https://inbo.github.io/tutorials/) for tags that have been used before)
- [X] I have added `categories` to the YAML header and my category tags are from the [list of category tags](https://github.com/inbo/tutorials/blob/master/static/list_of_categories)

## Previewing the pull request

Thanks to GitHub Actions, an artifact (=zip file) of the rendered website is automatically created for each pull request.
This provides a way to preview how these updates will look on the website, useful to contributors and reviewers.

### Instructions to preview the updated website

1) On the PR page, you can find a "details" link under "checks - On PR, build the site and ...". Go there, click on the top link in the left sidebar ("Summary"), and download the generated artifact at the bottom of the page.
2) Decompress it and make sure the target directory is called 'tutorials' (you may need to rename it)
3) From the parent directory (just above the `tutorials` folder you created/renamed), run `python -m http.server 8887`, _or_ launch the Google Chrome [Web Server app](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb) and point it at the parent directory.
4) Point your browser to http://localhost:8887/tutorials.
5) Review the updated website. As a contributor, you can push extra commits to update the PR. As a reviewer, you can accept/refuse/comment the PR.

**Note: for step 3, you can use any other simple HTTP server to serve the current directory if you don't have a Python 3 environment or Google Chrome available.**

### CODE REVIEW

File: content/tutorials/spatial_wms_services/index.md [hunk 1]
```diff
@@ -98,8 +102,12 @@ leaflet() %>%
   )
 ```
 
-![Leaflet map with the orthophoto mosaic as
-background](index_files/figure-gfm/unnamed-chunk-5-1.png)
+<figure>
```
Comment (ThierryO) [inline content/tutorials/spatial_wms_services/index.md:105]: Any reason for not keeping the original markdown code?

File: config.toml [hunk 2]
```diff
@@ -149,3 +149,9 @@ hrefTargetBlank = true
 lang = "en"
 languageName = "English"
 weight = 1
+
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
```
Comment (ThierryO) [inline config.toml:156]: do we need this?

Comment (florisvdh) [reply config.toml:156]: See https://github.com/inbo/tutorials/pull/324#issue-1800513065 and commit message of https://github.com/inbo/tutorials/commit/7db836f958760d0b79d3e04e38e477015f5115d9

File: content/tutorials/spatial_wms_services/index.md (hunk 1 above)
Comment (florisvdh) [reply content/tutorials/spatial_wms_services/index.md:105]: See [discussion above](https://github.com/inbo/tutorials/pull/324#issue-1800513065); it is current knitting/pandoc behaviour. I have tried to control this but did not find a solution.

**Review by hansvancalster — APPROVED** [review]
Thanks for taking care of this.

File: content/tutorials/spatial_wms_services/index.md (hunk 1 above)
Comment (ThierryO) [reply content/tutorials/spatial_wms_services/index.md:105]: Then @damianooldoni and @peterdesmet should review this.

**Review by damianooldoni — APPROVED** [review]
Nice work @florisvdh 💪 I didn't find a better way to solve this better than you did. 
I think we should carefully follow future developments of Hugo.

**Comment by florisvdh** [comment]
Thanks for reviewing @damianooldoni!

> I think we should carefully follow future developments of Hugo

Well, IIRC the current workflow implements a fixed Hugo version (see GHA workflow). That may be good for stability and perhaps theming aspects, but it may eventually raise incompatibility problems with other tooling. I cannot judge which is best in this case, either using latest or using an oldie.

**Pull request merged**